TWTValidation/Validators/Value Validators/TWTStringValidator.m

```diff
@@ -257,6 +263,59 @@ - (BOOL)validateValue:(id)value error:(out NSError *__autoreleasing *)outError
 
 #pragma mark
 
+@implementation TWTBoundedComposedCharacterLengthStringValidator
+
+- (BOOL)validateValue:(id)value error:(out NSError *__autoreleasing *)outError
+{
+    TWTValueValidator *stringValidator = [TWTValueValidator valueValidatorWithClass:[NSString class] allowsNil:self.allowsNil allowsNull:self.allowsNull];
```

Comment:
The superclass is TWTBoundedLengthValidator, to get the properties and `initWithMinimumLength:maximumLength:`. We know that the validation may be incorrect depending on the string passed in.